.circleci/config.yml

```diff
@@ -41,14 +42,68 @@ jobs:
             - sdk/javascript/node_modules
           key: v1-sdk-dependencies-{{ checksum "sdk/javascript/package.json" }}
       - run: mkdir ~/junit-oracle
-      - run: npx saddle compile
-      - run: docker-compose -p open-oracle build
-      - run: JEST_JUNIT_OUTPUT=~/junit-oracle/test-results.xml npx saddle test tests/*Test.js -- --ci --reporters=default --reporters=jest-junit
+      - run: JEST_JUNIT_OUTPUT=~/junit-oracle/test-results.xml script/test tests/*Test.js -- --ci --reporters=default --reporters=jest-junit
       - store_test_results:
           path: ~/junit-oracle
       - store_artifacts:
           path: ~/junit-oracle
 
+  test_coverage:
+    docker:
+      - image: circleci/node:13
+    working_directory: ~/repo
+    steps:
+      - run:
+        |
+          sudo wget https://github.com/ethereum/solidity/releases/download/v0.6.10/solc-static-linux -O /usr/local/bin/solc
+          sudo chmod +x /usr/local/bin/solc
```

Comment:
we already grabbed solc in `test-oracle`, do we have to re download it? does everything get wiped between jobs?

Reply:
yeah, they run totally separately